nav2_behavior_tree/plugins/action/follow_path_action.cpp

```diff
@@ -62,6 +64,8 @@ class FollowPathAction : public BtActionNode<nav2_msgs::action::FollowPath>
       {
         BT::InputPort<nav_msgs::msg::Path>("path", "Path to follow"),
         BT::InputPort<std::string>("controller_id", ""),
+        BT::InputPort<double>("distance_to_goal", "Distance to goal to stop translation"),
```

Comment:
Shouldn't this distance to goal be computed by the dynamic new goal to follow (e.g. it is set at the distance away from the obstacle)